README.md

```diff
@@ -145,6 +146,45 @@ The optional `revision` value is printed on the title page and in the header, if
 
 The manufacturer name, which must be specified in `system.yml` data document, is also show on the title page.
 
+## Audit Checklists
+
+We include several checklists for various standards. 
+These are used by the `rdm gap` command to check output documents for appropriate references to a given standard. 
+
+For example ISO62304 requires that the device be classified according to the hazard levels 
+it could present.
+This requirement occurs in section 4.3.a of the standard.
+All of the 62304 checklists will include the following:
+```
+62304:4.3.a Software safety classification: system class
```

Comment:
Perhaps it would be helpful to provide a more complete example that includes comments and includes?

Reply:
The checklists themselves provide examples. Plus there is the example given of making a master checklist.

Reply:
Great point! These ones will always be up to date. Maybe a link to the github files is worth adding?